build_defs/run_in_repo.bzl

```diff
@@ -21,40 +21,28 @@ def _write_shell_script(ctx, run_script):
     Returns the created script as a File object.
     """
 
-    # Extract the repo root directory from .bazelrc
-    if "repo_root" not in ctx.var:
```

Comment:
If repo_root isn't needed anymore, could you please also remove it from tools/setup?